index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML & CSS BASICS - #05</title>
+    <link href="./styles/reset.css" rel="stylesheet">
+    <link href="./styles/base.css" rel="stylesheet">
+    <link href="./styles/header.css" rel="stylesheet">
+    <link href="./styles/content.css" rel="stylesheet">
+    <link href="./styles/footer.css" rel="stylesheet">
+    
+</head>
+<body>
+
+
+    <header class="header row">
+
+        <nav class="header__container container">
+            <div class="header__logo cell">
+                <a href="#"><img src="./images/logo.svg" alt="Nova logo"></a>
+            </div>
+
+            <div class="header__navbar cell">
+                <ul>
+                    <li class="navbar__list"><a class="strong" href="#Team">The Team</a></li>
+                    <li class="navbar__list"><a class="strong" href="#Pricing">Pricing</a></li>
+                    <li class="navbar__list"><a class="strong" href="#Features">Features</a></li>
+                </ul>
+            </div>
+        </nav>
+
+        <div class="header__container container">
+            <article class="header__article cell">
+                <h1>Beautiful Free Nova template</h1>
+                <h4>
+                    A top notch premium quality<br>
+                    template for your next web<br>
+                     project. 
+                </h4>
+                <button class="strong"><a>Download FREE!</a></button>
+            </article>
+            
+            <article class="header__article cell">
+                <img src="./images/screen.png" alt="Screen">
```

Comment:
Tutaj grafika posiada "postrżępione" krawędzie - można to rozwiązać przez białe tło (takie jak ramka) dla elementu zewnętrznego.